app/preparation/fight.py

```diff
@@ -0,0 +1,23 @@
+def prepare(
+        list_of_participants: list,
+        application_dictionary: dict
+) -> None:
+    hortach = iter(application_dictionary.values())
+    for knight in list_of_participants:
+
+        equipment = next(hortach)
+        knight.protection = 0
+
+        for specimen in equipment["armour"]:
+            knight.protection += specimen["protection"]

```

Comment:
better to use `get`
```suggestion
        for specimen in equipment.get("armour", []):
            knight.protection += specimen.get("protection")
```
use `get` everywhere